src/Google/GlobalSiteTag.php

```diff
@@ -249,16 +257,44 @@ function gtag() { dataLayer.push(arguments); }
 
 			gtag('js', new Date());
 			gtag('set', 'developer_id.<?php echo esc_js( self::DEVELOPER_ID ); ?>', true);
-			gtag('config', '<?php echo esc_js( $ads_conversion_id ); ?>', {
-				'groups': 'GLA',
-				'send_page_view': false
-			});
+			<?php echo esc_js( $this->get_gtag_config( $ads_conversion_id ) ); ?>
```

Comment:
:construction: 
Escaping JS, results in syntax errors for escaped quotes
```suggestion
			<?php echo $this->get_gtag_config( $ads_conversion_id ); ?>
```